src/pages/Facility/services/inventory/externalSupply/deliveryOrder/DeliveryOrderShow.tsx

```
@@ -486,18 +486,37 @@ export function DeliveryOrderShow({
-                  {t("note")}
-                </label>
-                <p className="text-sm whitespace-pre-wrap">
-                  {deliveryOrder.note}
-                </p>
-              </div>
-            )}
+              {deliveryOrder.note && (
+                <div>
+                  <label className="text-sm font-medium text-gray-700">
+                    {t("note")}
+                  </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)